flink-table-store-core/src/main/java/org/apache/flink/table/store/file/utils/FileStorePathFactory.java

```diff
@@ -131,27 +129,4 @@ public String getPartitionString(BinaryRowData partition) {
     public String uuid() {
         return uuid;
     }
-
-    /** Cache for storing {@link DataFilePathFactory}s. */
-    public static class DataFilePathFactoryCache {
```

Comment:
We don't need this cache now, we can just use `pathFactory.bucketPath`.
Although it will convert the partition to string once at a time. The cost is not large.